gdk4/src/texture.rs

```diff
@@ -18,3 +21,47 @@ impl<O: IsA<Texture>> TextureExtManual for O {
         }
     }
 }
+
+impl Texture {
```

Comment:
Are all those variants thread-safe?

Reply:
According to the docs these functions are thread safe, ill double check that all of them are tho.

Reply:
Yes they all are.